src/themes/shared/slotRecipes/table.ts

```diff
@@ -264,78 +102,109 @@ export const tableSlotRecipe = defineSlotRecipe({
     variant: {
       line: {
         columnHeader: {
-          borderBottomWidth: '1px',
+          borderBottom: '1px',
+          borderColor,
         },
         cell: {
-          borderBottomWidth: '1px',
+          color: 'gray.900',
+          borderBottom: '1px',
+          borderColor,
         },
-        row: {
-          bg: 'bg',
+        caption: {
+          color: 'gray.700',
         },
       },
       outline: {
-        root: {
-          boxShadow: '0 0 0 1px {colors.border}',
-          overflow: 'hidden',
-        },
         columnHeader: {
-          borderBottomWidth: '1px',
+          borderBottom: '1px',
+          borderColor,
         },
-        header: {
-          bg: 'bg.muted',
+        cell: {
+          borderBottom: '1px',
+          borderColor,
         },
-        row: {
-          '&:not(:last-of-type)': {
-            borderBottomWidth: '1px',
-          },
+        header: {
+          bg: subtleColor,
         },
         footer: {
-          borderTopWidth: '1px',
+          borderTop: '1px',
+          borderColor,
         },
       },
     },
     size: {
       sm: {
-        root: {
-          textStyle: 'sm',
-        },
         columnHeader: {
-          px: '2',
-          py: '2',
+          px: 'xs',
+          py: 'xs',
+          textStyle: 'heading5',
         },
         cell: {
-          px: '2',
-          py: '2',
+          px: 'xs',
+          py: 'xs',
+          textStyle: 'body',
+        },
+        caption: {
+          px: 'xs',
+          py: 'xs',
+          textStyle: 'body',
+          fontSize: 'sm',
         },
       },
       md: {
-        root: {
-          textStyle: 'sm',
-        },
         columnHeader: {
-          px: '3',
-          py: '3',
+          px: 'md',
+          py: 'sm',
+          textStyle: 'heading4',
+          fontSize: 'sm',
         },
         cell: {
-          px: '3',
-          py: '3',
+          px: 'md',
+          py: 'sm',
+          textStyle: 'body',
+        },
+        caption: {
+          px: 'md',
+          py: 'sm',
+          textStyle: 'body',
         },
       },
-      lg: {
-        root: {
-          textStyle: 'md',
+    },
+  },
+  compoundVariants: [
```

Comment:
[Compound variants](https://www.chakra-ui.com/docs/theming/slot-recipes#compound-variants) is a way of overriding styles based on a combination of several variants